### ISSUE TITLE
Cannot install on Debian-based distro

### ISSUE THREAD

**Issue by jscottSwift** [IBM/ibmi-oss-issues#45]
I have a base Ubuntu image that has the ibm-iaccess driver installed.  However, running my build process has recently been failing.  I am using the commands [provided in the documentation](https://github.com/IBM/ibmi-oss-docs/blob/master/odbc/installation.md).

```
curl https://public.dhe.ibm.com/software/ibmi/products/odbc/debs/dists/1.1.0/ibmi-acs-1.1.0.list | tee /etc/apt/sources.list.d/ibmi-acs-1.1.0.list
apt-get update -y
```
The image successfully built on June 13 2023, but has been failing since July 1 2023.  See the attached screenshot from my successful build.  The output shows the process installing the libodbc2 and other dependencies successfully.

However, I am now getting the following error:

The following packages have unmet dependencies:  
ibm-iaccess:  
- Depends: libodbc1 but it is not installable  
- Depends: odbcinst1debian2 but it is not installable  
- Recommends: unixodbc but it is not going to be installed

It seems like something has changed that it is now requiring old, nonexistent dependencies.

![successbuild](https://github.com/IBM/ibmi-oss-issues/assets/107221910/64156877-b9e6-4315-a72c-f33536765abe)

**Comment by jscottSwift** [comment]
I'm using Ubuntu 23.10.  But I am open to other suggestions.  Attached is the full dockerfile and the output logs showing success and failures.

The successful one was done back in June by our GitLab CICD pipeline.  That one was also installing .NET because this part of a larger base image for our .NET apps that utilize the iaccess driver to query our AS400.

Thanks,

Jeff Scott

**Comment by kadler** [comment]
23.10 isn't even out yet, but you said it has been failing since July so presumably you were using a different image then. Were you using a different image back in June? We only support the LTS releases for Ubuntu (currently 20.04 and 22.04) BTW.

**Comment by kadler** [comment]
So it looks like libodbc1 and odbcinst1debian2 went away with 23.10 at least. The latest driver 1.1.0.28 has fixes for this so it can be installed on 24.04. It may or may not work on 23.10, but we only support LTS releases as mentioned previously.
